frontend/src/queries/nodes/DataVisualization/Components/Charts/LineGraph.tsx

```diff
@@ -188,10 +188,28 @@ export const LineGraph = (): JSX.Element => {
                     },
                     scaleID: hasLeftYAxis ? 'yLeft' : 'yRight',
                     value: cur.value,
+                    enter: (ctx) => {
+                        if (ctx.chart.options.plugins?.annotation?.annotations) {
+                            const annotations = ctx.chart.options.plugins.annotation.annotations as Record<string, any>
+                            if (annotations[`line${curIndex}`]) {
+                                annotations[`line${curIndex}`].label.content = `${cur.label}: ${cur.value}`
+                                ctx.chart.update()
```

Comment:
Question: Is the value going to be formatted for really large revenue out of the box?